pkg/chaining/registry.go

```diff
@@ -1,34 +1,28 @@
 package chaining
 
 import (
-	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/authentication"
 	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/configuration"
 	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/editing"
-	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/logout"
 	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/noop"
 	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/reading"
 	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/service"
 )
 
 const (
-	ServiceAuthenticateName    = "srv_auth_in"
 	ServiceReadIssueName       = "srv_read"
 	ServiceEditCustomFieldName = "srv_edit_field"
-	ServiceLogoutName          = "srv_auth_out"
 	ServiceUnknownName         = "srv_unknown"
 )
 
 var serviceRegistry = make(map[string]service.Service)
 
 func InitServiceRegistry() {
-	serviceRegistry[ServiceAuthenticateName] = &authentication.ServiceAuthenticateSession{}
 	serviceRegistry[ServiceReadIssueName] = &reading.ServiceReadIssue{}
 	serviceRegistry[ServiceEditCustomFieldName] = &editing.ServiceEditCustomField{}
-	serviceRegistry[ServiceLogoutName] = &logout.ServiceLogoutSession{}
 	serviceRegistry[ServiceUnknownName] = &noop.ServiceUnknown{}
 }
 
-func GetServicesChain(c configuration.Context, secured bool) []service.Service {
+func GetServicesChain(c configuration.Context) []service.Service {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported function GetServicesChain should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4777494)